server/SetupSocketForGame.ts

```
@@ -0,0 +1,263 @@
+          } else {
+            console.error(`Failed to update elapsed time in DB for game ${gameId}`);
+          }
+        } catch (error) {
+          console.error(`Error updating elapsed time in DB for game ${gameId}:`, error);
+        }
+
+        // Update money for all players in this game
+        const connectedSockets = Array.from(room);
+        for (const socketId of connectedSockets) {
+          const socket = gameNamespace.sockets.get(socketId);
+          if (socket && socket.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)